cornice_swagger/swagger.py

```diff
@@ -104,12 +109,18 @@ def _build_paths(self, ignore=['head', 'options'], **kwargs):
 
                 op = self._extract_operation_from_view(view, args)
 
-                # XXX: If tag not defined, try to guess it from path
-                default_tag = service.path.split("/")[1]
-                if 'tags' not in op:
-                    op['tags'] = [default_tag]
+                # If tag not defined and a default tag is provided
+                if 'tags' not in op and default_tags:
+                    if callable(default_tags):
+                        default_tags = default_tags(service)
```

Comment:
We check [right below](https://github.com/Cornices/cornice.ext.swagger/blob/147d03dfde9397c61044d5510b5677b17e9aedcb/cornice_swagger/swagger.py#L119) if tags was defined as a list, also testing if a raw value was provided correctly. Do you think we need additional verification? I also came to me that a list with items that are not strings should also raise an exception.